styles.css

```diff
@@ -86,6 +104,9 @@ body {
 /*  ---- start exercise 5 ----  */
 .tags {
   width: 800px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
```

Comment:
Try using ` justify-content: center;` instead.
Also, to remove the list bullets you can use ` list-style-type: none;`